python/tvm/tir/tensor_intrin/cuda.py

```diff
@@ -817,6 +916,12 @@ def wmma_sync_impl(a: T.handle, b: T.handle, c: T.handle) -> None:
     *get_wmma_sync_intrin(16, 16, 16, "int8", "int32", True),
 )
 
+WMMA_SYNC_8x8x32_s4s4s32_TRANS_INTRIN = "wmma_sync_8x8x32_s4s4s32_trans"
```

Comment:
"wmma_sync_8x8x32_s4s4s32" is missing.

Reply:
sub-byte tensor core only allows A in row major and B in col major

Reply:
Oh that's interesting!